acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueDiscardRpcAcceptanceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.clique;
+
+import tech.pegasys.pantheon.tests.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.pantheon.tests.acceptance.dsl.node.PantheonNode;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class CliqueDiscardRpcAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void shouldDiscardVotes() throws IOException {
+    final String[] initialValidators = {"miner1", "miner2"};
+    final PantheonNode minerNode1 =
+        pantheon.createCliqueMinerNodeWithValidators("miner1", initialValidators);
+    final PantheonNode minerNode2 =
+        pantheon.createCliqueMinerNodeWithValidators("miner2", initialValidators);
+    final PantheonNode minerNode3 =
+        pantheon.createCliqueMinerNodeWithValidators("miner3", initialValidators);
+    cluster.start(minerNode1, minerNode2, minerNode3);
+
+    minerNode1.execute(cliqueTransactions.createProposeRemove(minerNode2));
+    minerNode2.execute(cliqueTransactions.createProposeRemove(minerNode2));
+    minerNode1.execute(cliqueTransactions.createProposeAdd(minerNode3));
+    minerNode2.execute(cliqueTransactions.createProposeAdd(minerNode3));
+    minerNode1.execute(cliqueTransactions.createDiscard(minerNode2));
+    minerNode1.execute(cliqueTransactions.createDiscard(minerNode3));
+
+    minerNode1.waitUntil(wait.chainHeadHasProgressed(minerNode1, 2));
```

Comment:
It doesn't require blockchain work. I only put that in there so that if the discard failed and it didn't actually discard then the test would fail because the chain would have progressed and the proposals added in that case.